<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>calculator</class>
 <widget class="QMainWindow" name="calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>166</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="CountButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Count</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ResetButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>250</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
